homeassistant/components/met/weather.py

```diff
@@ -142,7 +143,7 @@ def name(self) -> str:
 
         return f"{DEFAULT_NAME}{name_appendix}"
 
-    @property
```

Comment:
This one can be done in the constructor, needs a separate PR

Reply:
#100334